src/server/commands/ccc.rs

```diff
@@ -21,10 +21,14 @@ where
     S::Metadata: storage::Metadata,
 {
     async fn execute(&self, args: CommandArgs<S, U>) -> Result<Reply, FTPError> {
-        let tx = args.tx.clone();
+        let mut tx: Sender<InternalMsg> = args.tx.clone();
         let session = args.session.lock().await;
         if session.cmd_tls {
-            spawn!(tx.send(InternalMsg::PlaintextControlChannel));
+            tokio02::spawn(async move {
```

Comment:
Its not unlikely that we can now remove some of these spawns, but for now its good that we keep it the same.